chapters/6-analysis.tex

```diff
@@ -79,25 +78,25 @@ \section{Constructing analysis datasets}
 test for plot-level productivity gains,
 and check if village characteristics are balanced.
 Having three separate datasets for each of these three pieces of analysis
-will result in much cleaner do-files than if they all started from 
-a single analysis dataset that constantly needs to be transformed.
+will result in cleaner and more efficient analytical code than if
+you started from a single analysis dataset and repeatedly transformed it.
 
 \subsection{Fitting construction into the data workflow}
 
 % Why construction is separate from data cleaning
-Construction is done after from data cleaning for two reasons.
-First, to clearly differentiate correction of data entry errors
-(necessary for all interactions with the data)
+Construction follows data cleaning for two reasons.
+First, this helps to clearly differentiate error corrections (necessary for all data uses)
 from creation of analysis indicators (necessary only for specific analyses).
-Second, to ensure that variable definitions are consistent across.
+Second, it helps to ensures that variable definitions are consistent across datasets.
 Construction can create many outputs combining many inputs,
-and, unlike in data cleaning, output datasets should be parsimonious rather than exhaustive.
+and, unlike in data cleaning,
+output datasets should be parsimonious rather than exhaustive.
```

Comment:
We should use a different word than `parsimonious`. I am not even sure what the meaning of this sentence is.